spec/spec_helper.rb

```diff
@@ -2,7 +2,6 @@
 
 require "rubocop-sorbet"
 require "rubocop/rspec/support"
-require "byebug"
```

Comment:
Should we add `require "debug/prelude"`, so that we can use breakpoints without requiring the debugger manually?

Reply:
Oh I didn't know that was a thing! Yeah let's do it.